paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentSheetScreen.kt

```diff
@@ -301,6 +303,7 @@ private fun PaymentSheetContent(
     }
 }
 
+@Suppress("LongMethod")
```

Comment:
Suppressing this, it's two lines over the limit and refactoring this function over that seems not worth it.

Reply:
+1. Want to remove the suppress, and do a `./gradlew detektbaseline` instead?